pkg/install/stack/types.go

```diff
@@ -40,7 +40,7 @@ type DeployOptions struct {
 	ChartsDirectory                   string
 	Logger                            *logrus.Entry
 	EnableCertManagerV2Migration      bool
-	EnableNginxIngressMigration       bool
+	EnableChartMigration              bool
```

Comment:
Please keep the "1 flag per migration" logic. Those flags are not meant to be convenient, but instead they should stop the admin and prevent them from making mistakes.

If we now introduce a "catch all" flag, the admin will just change their deployment script to include `--enable-chart-migrations` and forget about it. With the KKP release after that, we might ship some super critical migration that includes some downtime and the admin would be caught off guard and have the migration run without their consent.

To prevent this, the flags are there. If it's required, we might have 1 flag per chart, or even multiple (IMHO). ^^ I would have very little problem with requring `--enable-prometheus-migration --yes-really-migrate-prometheus --accept-monitoring-downtime=$(date)` if we had some super critical Prometheus migration and we really, really need the admin to read the release notes first.

The inconvenience of having to add yet another flag (temporarily) to the installer is insignificant compared to the safety win and peace of mind. :)